config/hooks/pre-commit-stub

```diff
@@ -9,5 +9,6 @@ REPO_ROOT_DIR="$(git rev-parse --show-toplevel)"
 PRE_COMMIT_SCRIPT="${REPO_ROOT_DIR}/config/hooks/pre-commit"
 
 if [ -f "$PRE_COMMIT_SCRIPT" ]; then
-    . "$PRE_COMMIT_SCRIPT"
+  # shellcheck source=pre-commit
+  . "$PRE_COMMIT_SCRIPT"
```

Comment:
Unfortunately, `. "$PRE_COMMIT_SCRIPT"` does not work any more, as that tries to `source` the script in `sh` (hence `bash` never runs).  We need just `"$PRE_COMMIT_SCRIPT"` instead, to respect the sha-bang there.

Reply:
Oops, good that you tested! Fixed in 4624c4d70050defcd2a2ecf59d2610562e1d0837.